backend/backends/graphite/graphite.go

```diff
@@ -85,6 +85,10 @@ type client struct {
 	legacyNamespace  bool
 }
 
+func (client *client) Run(ctx context.Context) error {
+	return client.sender.Run(ctx)
+}
```

Comment:
Maybe write an "integration" test that exercises the interface, that will prevent similar error in the future. Not sure if it's idiomatic but it's more robust.